### PR TITLE
Add basics 1

### PR DESCRIPTION
**Pull request by LucaAllmaier**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- currently, the copyright date is only 2019
- currently, the project only consists of the template
- currently, there is no LandingPage 

**What is the new behavior (if this is a feature change)?**
- The copyright dates in all files have been changed to 2019-2020.
- The output of unused components has been removed.
- A LandingPage has been created.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,36 +5,18 @@ import React from "react";
...
-/**
- * Footer: Global Footer
- * Navbar: Global navigation bar
- */
-import {
-  Footer,
-  Navbar,
-} from "./components/molecules";
-// Routes
-import Routes from "./Routes";
```
Comment (Aichnerc) [inline src/App.js:17]: Remove all unused files like Routes.js and all components.

File: src/Routes.js [hunk 2]
```diff
@@ -8,16 +8,14 @@ import { Route, Switch } from "react-router-dom";
...
-} from "./components/pages";
+import { HomePage } from "./components/pages";
 
 class Routes extends React.Component {
   render() {
     return (
       <Switch>
         <Route exact path="/" component={HomePage} />
-          
+
```
Comment (schettn) [inline src/Routes.js]: You should remove this newline.

**Review by Aichnerc — CHANGES_REQUESTED** [review]
Please remove all unused files and comonents.

**Review by Aichnerc — APPROVED** [review]
LGTM 👀

File: src/components/pages/LandingPage/LandingPage.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+.container {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  overflow: hidden;
+  z-index: -1;
+}
```
Comment (schettn) [inline src/components/pages/LandingPage/LandingPage.scss:34]: Licence missing

**Review by schettn — CHANGES_REQUESTED** [review]
Should be ok after a adjustment

File: src/Routes.js (hunk 2 above)
Comment (LucaAllmaier) [reply src/Routes.js]: The whole file has been removed in a later commit

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Comment by kleberbaum** [comment]
In the PR message this:

> **What is the current behavior? (link to any open issues here)**
> 
>   * currently, the copyright date is only 2019
> 
>   * currently, the project only consists of the template
> 
>   * currently, there is no LandingPage
> 
> 

Should be changed to something like this:

**What is the current behavior? (link to any open issues here)**
   - Currently the copyright covers only 2019.
   - Currently the project only consists of the template.
   - Currently there is no LandingPage.

**Review by schettn — APPROVED** [review]
LGTM

**Pull request merged**